agent/consul/config_endpoint_test.go

```diff
@@ -310,63 +310,88 @@ func TestConfigEntry_Get_BlockOnNonExistent(t *testing.T) {
 	}
 
 	_, s1 := testServerWithConfig(t)
```

Comment:
I refactored this test to follow the same flow as the others I just added for consistency.